File: 30-days-of-code/day2/Solution.java
import java.io.*;
import java.util.*;
import java.io.*;

public class Solution {
    public static long calculateTotalCost(double mealCost, int tipPercent, int taxPercent) {
        double tipCost = mealCost * (tipPercent / 100.0);
        double taxCost = mealCost * (taxPercent / 100.0);
        return Math.round(mealCost + tipCost + taxCost);
    }
        
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble();
        int tipPercent = scan.nextInt();
        int taxPercent = scan.nextInt();

        long result = calculateTotalCost(mealCost, tipPercent, taxPercent);
        System.out.println(result);
    }
}

File: 30-days-of-code/day2/solution.py
import math
import os


def calculate_total_cost(meal_cost, tip_percent, tax_percent):
    tip_cost = meal_cost * (tip_percent / 100.0)
    tax_cost = meal_cost * (tax_percent / 100.0)
    return round(meal_cost + tip_cost + tax_cost)


def main():
    meal_cost = float(input())
    tip_percent = int(input())
    tax_percent = int(input())
    result = calculate_total_cost(meal_cost, tip_percent, tax_percent)
    print(result)


if __name__ == "__main__":
    main()
